cmd/control/os.go

```diff
@@ -324,3 +328,10 @@ func getUpgradeURL() (string, error) {
 	cfg := config.LoadConfig()
 	return cfg.Rancher.Upgrade.URL, nil
 }
+
+func isUpgradeAvailable() bool {
+	if _, err := os.Stat("/proc/1/root/boot/global.cfg"); err == nil || os.IsExist(err) {
```

Comment:
Can we make a common func about `/proc/1/root/boot/global.cfg`?
I can see this condition in other places.